sky/execution.py

```diff
@@ -975,6 +976,8 @@ def serve_up(
     """
```

Comment:
add docstring for `controller_resources`?

Reply:
Done. Thanks!